README.md

```diff
@@ -52,11 +52,11 @@ end
 # You can either provide a hash of parameters to the initializer
 assessment = Minfraud::Assessments.new(
   device: {
-    ip_address: '1.2.3.4.5'
+    ip_address: '1.2.3.4'
```

Comment:
We should probably change this to use an IP from a range reserved for documentation or make it one of the IPs that Tawa provided. This reminds me that I should probably do that myself on the Python `geoip2` PR.

Reply:
Yes, good point! Thank you.

Reply:
I edited the commit where I changed the IP. Also I added one new commit for something unrelated I noticed!